docs/integration.md

```diff
@@ -184,8 +185,9 @@ CRS-aware! (just a warning is emitted below).
 ds_geo_wgs84.sel(lat=70)
 ```
 
-Since ``GeoIndex`` also implements the ``_proj_get_crs`` method it is possible
-to get the CRS from the "lat" coordinate like so:
+``GeoIndex`` has a ``crs`` property (as required by
+{class}`~xproj.ProjIndexMixin`), which is possible to access also via the
+``proj`` accessor like so:
 
 ```{code-cell} ipython3
 ds_geo_wgs84.proj("lat").crs
```

Comment:
At this point, there is no difference between `ds_geo_wgs84.geo.crs`, `ds_geo_wgs84.proj.crs`, and  `ds_geo_wgs84.proj("lat").crs` correct?

Reply:
> For example, you can assign_crs without warning or error, but then as soon as you try to access one of the .crs values you hit [a ValueError]

(the ValueError raised may actually be useuful to prevent nasty bugs due to such inconsistent situations... but I agree that it is preferable to raise or warn sooner)

Reply:
Opened #29